src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -1773,7 +1773,7 @@
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				GCC_DYNAMIC_NO_PIC = NO;
 				GCC_NO_COMMON_BLOCKS = YES;
-				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_OPTIMIZATION_LEVEL = g;
```

Comment:
This is the same "enable -Og for debug builds" thing as above, right?  Why do we want that?

Reply:
I can take these back out... It's "the same" but not redundant because the setting doesn't propagate between the build systems.